CircleCI-Public/circleci-yaml-language-server
# Adds missing Windows resource classes

# Description

> Adds machine executor resource classes for Windows resources.

# Implementation details

> Adding `windows.medium` and other sized resource classes to the list of valid `resource_class` enum values.

# How to validate

> Add a Windows resource class to a Machine executor, such as `windows.medium`, and check that it validates as expected.
